mkdocs.yml

```diff
@@ -26,6 +26,7 @@ nav:
       - Legacy: api/legacy.md
       - api/pgstac.md
   - drawbacks.md
+  - Release Notes: whatsnew.md
```

Comment:
👍🏼 to keeping the software release notes in their own file, to help disambiguate from spec changes.